### ISSUE TITLE
Iris segmentation

### ISSUE THREAD

**Issue by ZuoyanL** [Debatrix/UniNet-Pytorch#3]
I noticed that the results after iris segmentation were inaccuracy. Could you please tell me how can I get the excellent segmentation results with the scripts.

**Comment by Debatrix** [comment]
You may need to finetune this model on your dataset. Unfortunately, I just reproduced this model and converted the parameters from caffe to pytorch, and the author did not release the training code.

**Comment by ZuoyanL** [comment]
> You may need to finetune this model on your dataset. Unfortunately, I just reproduced this model and converted the parameters from caffe to pytorch, and the author did not release the training code.

Do you know how to get the accuracy IRIS segmentation results? You know, I can't get the pupils using the scripts from either yours or the author's.

**Comment by Debatrix** [comment]
> Do you know how to get the accuracy IRIS segmentation results? You know, I can't get the pupils using the scripts from either yours or the author's.

The most accurate method is to segment manually. 😀 As far as I know, there are relatively few open source codes in this field, you can try Osiris, [TVM](https://www4.comp.polyu.edu.hk/~csajaykr/tvmiris.htm), [IrisParseNet](https://github.com/xiamenwcy/IrisParseNet) and [IrisSegBenchmark](https://github.com/xiamenwcy/IrisSegBenchmark)

**Comment by ZuoyanL** [comment]
> > Do you know how to get the accuracy IRIS segmentation results? You know, I can't get the pupils using the scripts from either yours or the author's.
> 
> The most accurate method is to segment manually. 😀 As far as I know, there are relatively few open source codes in this field, you can try Osiris, [TVM](https://www4.comp.polyu.edu.hk/~csajaykr/tvmiris.htm), [IrisParseNet](https://github.com/xiamenwcy/IrisParseNet) and [IrisSegBenchmark](https://github.com/xiamenwcy/IrisSegBenchmark)

Yeah, I see. Thank you very much. by the way, do you use wechat?
